vulture/core.py

```diff
@@ -27,26 +27,19 @@
 from __future__ import print_function
 
 import ast
-import codecs
 from fnmatch import fnmatchcase
 import optparse
 import os
 import pkgutil
 import re
 import sys
-import tokenize
 
+from vulture.utils import (ENCODING_REGEX, format_path, LoggingList, read_file,
+                           VultureInputException)
 from vulture.lines import estimate_lines
```

Comment:
I'd prefer to just `from vulture import lines` and on a separate line `from vulture import utils` and use e.g. `utils.format_path` in the code. This makes it clear where the function comes from on first sight.